### PR TITLE
Upgrade terraform-provider-yandex to v0.69.0

### LINKED ISSUE #24: Upgrade to v0.67.0 of the Yandex Terraform Provider

### LINKED ISSUE #27: Upgrade to v0.68.0 of the Yandex Terraform Provider

### LINKED ISSUE #29: Upgrade to v0.69.0 of the Yandex Terraform Provider

### PR DESCRIPTION
**Pull request by jkodroff**
Fixes: #29
Fixes #27 
Fixes #24

### CODE REVIEW

**Comment by jkodroff** [comment]
Well that doesn't look right...

**Comment by jkodroff** [comment]
@stack72 Maybe we should do these as 2 separate upgrades, since 0.67 brings in TF Plugin SDK v2.  What do you think?

File: CHANGELOG.md [hunk 1]
```diff
@@ -37,3 +39,166 @@ _(none)_
...
+
+
+
+
+
+
+        Press enter to move this card
+    
+
+
```
Comment (guineveresaenger) [inline CHANGELOG.md]: something happened here but also you might look into a rebase for this file in general

File: provider/go.mod [hunk 2]
```diff
@@ -4,12 +4,13 @@ go 1.16
 
 replace (
 	github.com/hashicorp/go-getter => github.com/hashicorp/go-getter v1.4.0
+	github.com/hashicorp/terraform-plugin-sdk/v2 => github.com/pulumi/terraform-plugin-sdk/v2 v2.0.0-20210629210550-59d24255d71f
```
Comment (guineveresaenger) [inline provider/go.mod:7]: Is having a forked provider new?

**Review by guineveresaenger — CHANGES_REQUESTED** [review]

File: provider/go.mod (hunk 2 above)
Comment (jkodroff) [reply provider/go.mod:7]: That's not a provider. That's our version of the TF SDK. This dependency is common to all providers.

**Review by guineveresaenger — APPROVED** [review]
When the run finishes, :shipit:

File: CHANGELOG.md (hunk 1 above)
Comment (jkodroff) [reply CHANGELOG.md]: Fixed. Nice catch!

**Review by guineveresaenger — CHANGES_REQUESTED** [review]
I think we should address the linter complaints, there's some trailing `Ptr*` references it seems

**Comment by jkodroff** [comment]
SDK linting is failing due to a bug in codegen. @stack72 is going to update the bridge, after which we can update the bridge dependency here and the build should pass.

**Review by guineveresaenger — APPROVED** [review]

**Pull request merged**